fpdc/settings.py

```diff
@@ -82,8 +82,11 @@
 
 DATABASES = {
     "default": {
-        "ENGINE": "django.db.backends.sqlite3",
-        "NAME": os.path.join(BASE_DIR, "db.sqlite3"),
+        "ENGINE": "django.db.backends.postgresql",
```

Comment:
This may require us to have a unittest-specific settings files in the future, but it's fine I guess.
Or we could write a `settings_container.py` file that would set the container-specific variables.

Reply:
Yes I reorganized the settings files under a settings folder so we can be more modular as advice by @subpop.